gui/public/i18n/vi/translation.ftl

```diff
@@ -365,156 +365,156 @@ settings-osc-vrchat-network-trackers-elbows = Khuỷu tay
+onboarding-connect_tracker-back = Quay lại cài đặt Wi-Fi
 onboarding-connect_tracker-title = Kết nối tracker
-onboarding-connect_tracker-description-p0 = Đến giờ đoạn hay nhất, kết nối tất cả tracker!
-onboarding-connect_tracker-description-p1 = Đơn thuần kết nối tất cả tracker chưa kết nối, qua cổng USB
-onboarding-connect_tracker-issue-serial = Mình có vấn đề kết nối!
+onboarding-connect_tracker-description-p0 = Cập nhật thông tin Wi-Fi cho tracker
+onboarding-connect_tracker-description-p1 = Hãy kết nối các tracker chưa thiết lập qua cổng USB
+onboarding-connect_tracker-issue-serial = Có vấn đề với việc kết nối? Kiểm tra thông tin qua cổng Serial
 onboarding-connect_tracker-usb = USB Tracker
 onboarding-connect_tracker-connection_status-connecting = Đang gửi thông tin Wi-Fi
-onboarding-connect_tracker-connection_status-connected = Đã kết nối Wi-Fi
-onboarding-connect_tracker-connection_status-error = Không thể kết nối Wi-Fi
+onboarding-connect_tracker-connection_status-connected = Đã kết nối đến Wi-Fi
+onboarding-connect_tracker-connection_status-error = Không thể kết nối đến Wi-Fi
 onboarding-connect_tracker-connection_status-start_connecting = Đang tìm tracker
-onboarding-connect_tracker-connection_status-handshake = Đã kết nối với máy chủ
+onboarding-connect_tracker-connection_status-handshake = Đã kết nối đến máy chủ SlimeVR
 # $amount (Number) - Amount of trackers connected (this is a number, but you can use CLDR plural rules for your language)
 # More info on https://www.unicode.org/cldr/cldr-aux/charts/22/supplemental/language_plural_rules.html
 # English in this case only has 2 plural rules, which are "one" and "other",
 # we use 0 in an explicit way because there is no plural rule in english for 0, so we directly say
 # if $amount is 0 then we say "No trackers connected"
 onboarding-connect_tracker-connected_trackers = { $amount ->
-    [0] Không trackers
+    [0] Không có tracker
     [one] 1 tracker
-    *[other] { $amount } trackers
+    *[other] { $amount } tracker
 } đã giao
-onboarding-connect_tracker-next = Tôi đã kết nối tất cả tracker
+onboarding-connect_tracker-next = Đã kết nối với tất cả tracker
 
 ## Tracker assignment setup
-onboarding-assign_trackers-back = Quay lại tra thông tin Wi-Fi
-onboarding-assign_trackers-title = Giao tracker
-onboarding-assign_trackers-description = Hãy chọn tracker nào nằm ở đâu. Chọn vị trí bạn muốn giao tracker cho
+onboarding-assign_trackers-back = Quay lại cài đặt Wi-Fi
+onboarding-assign_trackers-title = Gán tracker
+onboarding-assign_trackers-description = Chọn vị trí bạn muốn gán tracker bằng cách nhấn vào tên bộ phận muốn gán và chọn tracker
 # Look at translation of onboarding-connect_tracker-connected_trackers on how to use plurals
 # $assigned (Number) - Trackers that have been assigned a body part
 # $trackers (Number) - Trackers connected to the server
 onboarding-assign_trackers-assigned = { $assigned } of { $trackers ->
```

Comment:
there is an ``of`` in here, might have accidentally skipped it

Reply:
I think you are right, been finding the missing `of` for a while but it might have been this